lib/compat/wordpress-6.2/html-api/class-wp-html-tag-processor.php

```diff
@@ -1732,7 +1732,7 @@ public function get_attribute( $name ) {
 	 * @param string $prefix Prefix of requested attribute names.
 	 * @return array|null List of attribute names, or `null` when no tag opener is matched.
 	 */
-	function get_attribute_names_with_prefix( $prefix ) {
+	public function get_attribute_names_with_prefix( $prefix ) {
```

Comment:
I didn't see this patched in 6.2, so I'm not sure we're supposed to backport it here.

Reply:
Ah, good point, thanks! I was confused a bit on which file to patch, so edited both just in case 🙂 Fixed in d95ae25.

Reply:
> confused a bit on which file to patch

this is something I'm constantly confused about. asked about it just a day ago on the HTML Processor backport, which is why I had it on my mind to point out here 😄